dts/arm/st/f4/stm32f427.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f4/stm32f429.dtsi>
```

Comment:
Okay, yes so on this note. The thought did come to mind to either base 27 on the already present 29 or redo 27 as if from scratch and then base 29 on it?

Also in the case what happens to the Licences headers?

Reply:
>  redo 27 as if from scratch and then base 29 on it

Yes

> Also in the case what happens to the License headers?

They'll be both Linaro, so that's just a matter of year. You can move both to 2020!